app/components/facet_input_component/organisation_multi_select_component.rb

```diff
@@ -0,0 +1,27 @@
+class FacetInputComponent::OrganisationMultiSelectComponent < ViewComponent::Base
```

Comment:
Seems a reasonable call to make right now 👍 
Do we have a backlog item to address the non-conforming organisations field?